packages/mobile/src/screens/search-screen/SearchItem.tsx

```diff
@@ -142,14 +138,12 @@ export const SearchItemTrack = (props: SearchItemProps) => {
 export const SearchItemCollection = (props: SearchItemProps) => {
   const { searchItem, onPress } = props
   const { id } = searchItem
-  const { data: playlist, status } = useGetPlaylistById({
-    playlistId: id
-  })
+  const { data: playlist, status } = useCollection(id)
   const navigation = useNavigation()
 
   const { data: playlistUser } = useUser(playlist?.playlist_owner_id)
 
-  if (status === Status.LOADING) return <SearchItemSkeleton />
+  if (status === 'pending') return <SearchItemSkeleton />
```

Comment:
nit: can unpack `isPending` from the useCollection result

Reply:
oh interesting.... yeah down